proposals/simd/SIMD.md

```diff
@@ -665,6 +665,11 @@ natural alignment.
 
 Load a `v128` vector from the given heap address.
 
+```python
+def S.load(memarg):
+    return S.New(memory.bytes[memarg.start:memarg.start + 16])
```

Comment:
Dumb suggestion, let's split `S.new()` and the actual loading of memory into S. We seem to have a style going on here where we do:
```
result = S.New()
for i in ???:
  do operation
```

So maybe this can look like
```
def S.load(memarg):
  result = S.new()
  for i in range(16):
    result[i] = memory.bytes[memarg.start + i]
```

Otherwise we might want to define what `S.New(*args)` means?

Reply:
I agree, maybe not the best idea to pass memory to `New`, will change that. It looks like other operations use `result[i]` for individual lanes, I am thinking of using something like `result.bytes[]` for binary representation of the vector value.

Reply:
Decided to use `from_bytes` for instantiating `S`, this works with some of Python types.